### PR TITLE
chore(ci): migrate to gh actions

### PR DESCRIPTION
**Pull request by hongaar**
Travis has almost no active development anymore, GitHub Actions is incredibly powerful. It also integrates better in the GH interface (for obvious reasons, it's GH's own product).

### CODE REVIEW

**Comment by AdrieanKhisbe** [comment]
> Travis has almost no active development anymore,

:thinking:
Kindaa bold statement :wink:

File: package.json [hunk 1]
```diff
@@ -76,4 +77,4 @@
   "files": [
     "dist"
   ]
-}
+}
```
Comment (AdrieanKhisbe) [inline package.json]: M'y newline :sob:

File: .travis.yml [hunk 2]
```diff
@@ -1,12 +0,0 @@
-language: node_js
```
Comment (AdrieanKhisbe) [inline .travis.yml:1]: Please mâke me live :pray:

**Comment by AdrieanKhisbe** [comment]
As I said in #20 I prefer we preserve thé Travis config for thé moment.
They Can perfectly coexist.

Also, why no coverage by default? Any readon in particular?

**Review by AdrieanKhisbe — COMMENTED** [review]
As I said in #20 I prefer we preserve thé Travis config for thé moment.
They Can perfectly coexist.

Also, why no coverage by default? Any readon in particular?

**Review by Ore4444 — APPROVED** [review]

**Comment by hongaar** [comment]
> Kindaa bold statement 😉

Sure, it's a hyperbole. But it's not out of thin air. Travis has seen no major features this past year, while competition is constantly moving...

**Comment by Ore4444** [comment]
The community is moving away from Travis.
I'm for trying out GH actions because it will allow us easier collaboration over the CI config in GitHb, in favor of manually editing a config file. 
@AdrieanKhisbe Any reasons why not to? What makes you see it's premature?

**Comment by AdrieanKhisbe** [comment]
> The community is moving away from Travis.
Honestly It's far too soon to say it's a shared fact without sharing any metrics, insight or any piece of proof.

And you don't change everydays config of the CI. (let aside the fact as I was the one the one that takes of most of the changes.

And just to be clear, my point is **NOT** "don't try github action", **IT'S** "don't kill travis now"

**Comment by AdrieanKhisbe** [comment]
@Ore4444  it's not the ~same~ first time I said exactly this on this specific subject.

👉  https://github.com/vorpaljs-reforged/vorpal/pull/20#pullrequestreview-319441068

And to be frank, it's starting to bore/annoy me a bit

**Comment by hongaar** [comment]
Not too important right now, closing in favour of more pressing issues.

**Pull request closed without merging**